tests/test_live.py

```diff
@@ -0,0 +1,45 @@
+import lomond
```

Comment:
This tests connects to a live websocket server.

This is actually a very good test, although relying on an external server makes be nervous.

It might be better to roll our own echo server for this test, but that's more effort. One for a later date.